reflex/utils/format.py

```diff
@@ -449,6 +450,10 @@ def format_state(value: Any) -> Any:
     if isinstance(value, go.Figure):
         return json.loads(to_json(value))["data"]  # type: ignore
 
+    # Convert datetime object to str.
+    if isinstance(value, datetime):
```

Comment:
while we're doing this anyway, is there a reason to not also support `date`, `time` and `timedelta`?

Reply:
Sure should I convert them to str?